src/view.js

```diff
@@ -190,7 +190,7 @@ const View = Backbone.View.extend({
   },
 
   _getImmediateChildren() {
-    return _.chain(this.getRegions())
+    return _.chain(this._getRegions())
```

Comment:
@paulfalgout do you see an issue with this?  Maintains functionality before auto-render was introduced.